test/verify/check-login

```diff
@@ -145,6 +145,13 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
             b.open("/system")
             b.wait_not_visible("#option-group")
 
+            # test login with tcsh
+            if not m.image.startswith("rhel") and not m.image.startswith("centos"):  # no tcsh in RHEL
+                m.execute("sed -r -i.bak '/^admin:/ s_:[^:]+$_:/bin/tcsh_' /etc/passwd")
+                b.login_and_go()
+                b.logout()
+                m.execute("mv /etc/passwd.bak /etc/passwd")
```

Comment:
Ah, this is tricky. This is a @nondestructive test, so it needs to restore /etc/passwd also on failure, but it actually does since all non-destructive tests explicitly restore /etc/passwd, as one of a number of special cases. So this is exactly correct, and I guess I am just not enough used to non-destructive tests...